docs/working-with-projects/working-with-images/managing-helm-charts.md

```diff
@@ -150,3 +152,54 @@ helm install --ca-file=ca.crt --username=admin --password=Passw0rd --version 0.1
 
 For other more helm commands like how to sign a chart, please refer to the [helm doc](https://docs.helm.sh/helm/#helm).
 
+
+## Manage Helm Charts with the OCI-compatible registry of Harbor
+
+Helm 3 support to pull chart from a OCI-compatible registry, and push chart to a OCI-compatible registry. About how to install Helm CLI, please refer [install helm](https://helm.sh/docs/intro/install/). Run command `helm version` to make sure the version of Helm CLI is v3.0.0+.
+
+```sh
+helm version
+
+#version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
+```
+
+### Login to the OCI-compatible registry of Harbor
+
+Before pull/push helm charts with the OCI-compatible registry of Harbor, Harbor should be logged with `helm registry login` command.
+
+```sh
+helm registry login xx.xx.xx.xx
+```
+
+{{< note >}}
+The CA file used by the Harbor is necessary to be trusted in the system due to an issue in Helm.
```

Comment:
due to an issue in Helm upstream

Reply:
According to this [issue](https://github.com/helm/helm/issues/6324), currently helm does not have option to configuration ca file for the self-signed certificate registry. To make helm 3 work with self-signed certificate harbor, the ca file used by the harbor must be trusted in the system level, otherwise pulling/pushing will be failed.